src/lib/output/themes/default/partials/member.declaration.tsx

```diff
@@ -29,12 +29,44 @@ export function memberDeclaration(context: DefaultThemeRenderContext, props: Dec
 
     const visitor = { reflection: renderTypeDeclaration };
 
+    /** Fix for #2717. If type is the same as value the type is omited */
+    function shouldRenderType() {
+        if (props.type && props.type.type === "literal") {
+            const typeObject = props.type.toObject();
+            const value = typeObject.value;
+            if (!value) {
+                // should be unreachable
+                return true;
+            }
+            if (typeof value === "object") {
+                return true;
+            }
+            const reflectionTypeString: string = value.toString();
+            let defaultValue = props.defaultValue!;
+            if (defaultValue) {
+                // If the default value is string and it's wrapped in ' in the code, the value is wrapped in " and vice-versa
```

Comment:
Ew... so this breaks if there are any escape sequences in the string... I'm okay ignoring that edge case for now.